cumulusci.yml

```diff
@@ -521,7 +555,15 @@ plans:
                         name: K-12 Architecture Kit Metadata and Configuration
             6:
                 task: deploy_k12_app
-                
+
+    picklists:
+        slug: picklists
+        title: Install K-12 Architecture Kit picklist values
+        tier: secondary
```

Comment:
We might want to use `tier: additional` for this, but that can be decided on review on Staging.